vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/exporter/ZephyrExporter.java

```diff
@@ -60,20 +64,30 @@ public void exportResults() throws IOException
         ZephyrConfiguration configuration = zephyrFacade.prepareConfiguration();
         for (TestCase testCase : testCasesForImporting)
         {
-            createNewTestExecution(testCase, configuration, objectMapper);
+            exportTestExecution(testCase, configuration);
         }
     }
 
-    private void createNewTestExecution(TestCase testCase, ZephyrConfiguration configuration, ObjectMapper objectMapper)
-            throws IOException
+    private void exportTestExecution(TestCase testCase, ZephyrConfiguration configuration) throws IOException
     {
         JiraEntity issue = jiraFacade.getIssue(testCase.getKey());
         ZephyrExecution execution = new ZephyrExecution(configuration, issue.getId(), testCase.getStatus());
-        String createExecution = objectMapper.writeValueAsString(execution);
-        int executionId = zephyrFacade.createExecution(createExecution);
+        int executionId;
 
-        String executionBody = objectMapper.writeValueAsString(new ExecutionStatus(
+        if (zephyrExporterProperties.getUpdateExecutionStatusesOnly())
+        {
+            executionId = zephyrFacade.findExecutionId(issue.getId());
```

Comment:
shouldn't the export fail if test execution is not found?

Reply:
No, because it made for setup statuses for only created executions (in allure them can be more than created in jira)

Reply:
ok, then maybe some logging makes sense?